README.md

```diff
@@ -1 +1,3 @@
-# api-gateway
\ No newline at end of file
+# api-gateway
+this is the api-gateway that's connected to birthday-service through grpc.
+made by yarin.
```

Comment:
Add a link to the birthday-service so it is clear

Reply:
done